trigger/task.go

```diff
@@ -703,8 +703,11 @@ func (t *taskTriggers) taskRuntimeChange(sub *event.Subscription) (*notification
 	return t.generate(sub, fmt.Sprintf("changed in runtime by %.1f%% (over threshold of %s%%)", percentChange, percentString), "")
 }
 
-func isFailedTaskStatus(status string) bool {
-	return status == evergreen.TaskFailed || status == evergreen.TaskSystemFailed || status == evergreen.TaskTestTimedOut
+// isValidFailedTaskStatus only matches task statuses that should be triggered for failure.
+// For example, it excludes  setup failures.
+func isValidFailedTaskStatus(status string) bool {
```

Comment:
I feel like the addition of the word `valid` here doesn't really help me understand better what the function does. I personally feel like isFailedTaskStatus with the comment you added feels sufficient to me. 
But I don't feel strongly either way so you decide :)

Reply:
I kind of just want to distinguish it from the non-private-trigger function "IsFailedTaskStatus" but yeah i recognize it doesn't add much

Reply:
Oh, that makes sense